src/controllers/update/confirmation.controller.ts

```diff
@@ -17,8 +19,24 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
     const appData: ApplicationData = getApplicationData(req.session);
     const referenceNumber = appData[Transactionkey];
 
+    // It's necessary to do this check (and save the result) before deleting the application data
```

Comment:
update the comment to say why - say app data used by isRemoveJourney

Reply:
Done.